mlir/utils/performance/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ set(PERFORMANCE_SCRIPTS
   tuningRunner.py
 )
 
-list(TRANSFORM PERFORMANCE_SCRIPTS PREPEND "${CMAKE_CURRENT_SOURCE_DIR}/")
```

Comment:
Why remove the quotes? Won't they be needed in case the directory contains a space?